From 5e00f464c546debf9164f24c6d648623f7ee5946 Mon Sep 17 00:00:00 2001
From: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
Date: Wed, 17 Mar 2010 14:56:24 -0500
Subject: [PATCH 13/37] Fix relink targets for silent rules

Add $(AM_V_at) to all relink make targets to silence them when automake
silent rules are in use.

Signed-off-by: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
Reviewed-by: Alan Coopersmith <alan.coopersmith@sun.com>
Reviewed-by: Julien Cristau <jcristau@debian.org>
---
 Makefile.am                 |    2 +-
 hw/Makefile.am              |    2 +-
 hw/dmx/Makefile.am          |    2 +-
 hw/kdrive/Makefile.am       |    2 +-
 hw/kdrive/ephyr/Makefile.am |    2 +-
 hw/kdrive/fake/Makefile.am  |    2 +-
 hw/kdrive/fbdev/Makefile.am |    2 +-
 hw/vfb/Makefile.am          |    2 +-
 hw/xfree86/Makefile.am      |    2 +-
 hw/xnest/Makefile.am        |    2 +-
 hw/xwin/Makefile.am         |    2 +-
 11 files changed, 11 insertions(+), 11 deletions(-)

diff --git xorg-server-1.7.99.902/Makefile.am xorg-server-1.7.99.902/Makefile.am
index 667c0de..8b7a2c8 100644
--- xorg-server-1.7.99.902/Makefile.am
+++ xorg-server-1.7.99.902/Makefile.am
@@ -94,4 +94,4 @@ DIST_SUBDIRS = \
 
 # gross hack
 relink: all
-	$(MAKE) -C hw relink
+	$(AM_V_at)$(MAKE) -C hw relink
diff --git xorg-server-1.7.99.902/hw/Makefile.am xorg-server-1.7.99.902/hw/Makefile.am
index 7409906..92b007a 100644
--- xorg-server-1.7.99.902/hw/Makefile.am
+++ xorg-server-1.7.99.902/hw/Makefile.am
@@ -38,4 +38,4 @@ SUBDIRS =			\
 DIST_SUBDIRS = dmx xfree86 vfb xnest xwin xquartz kdrive
 
 relink:
-	for i in $(SUBDIRS) ; do $(MAKE) -C $$i relink ; done
+	$(AM_V_at)for i in $(SUBDIRS) ; do $(MAKE) -C $$i relink ; done
diff --git xorg-server-1.7.99.902/hw/dmx/Makefile.am xorg-server-1.7.99.902/hw/dmx/Makefile.am
index 1e3e5da..cdee586 100644
--- xorg-server-1.7.99.902/hw/dmx/Makefile.am
+++ xorg-server-1.7.99.902/hw/dmx/Makefile.am
@@ -114,4 +114,4 @@ SUFFIXES = .$(APP_MAN_SUFFIX) .man
 EXTRA_DIST = $(appman_PRE)
 
 relink:
-	rm -f Xdmx$(EXEEXT) && $(MAKE) Xdmx$(EXEEXT)
+	$(AM_V_at)rm -f Xdmx$(EXEEXT) && $(MAKE) Xdmx$(EXEEXT)
diff --git xorg-server-1.7.99.902/hw/kdrive/Makefile.am xorg-server-1.7.99.902/hw/kdrive/Makefile.am
index 9e6c18e..55d940f 100644
--- xorg-server-1.7.99.902/hw/kdrive/Makefile.am
+++ xorg-server-1.7.99.902/hw/kdrive/Makefile.am
@@ -27,4 +27,4 @@ SUBDIRS =			\
 DIST_SUBDIRS = fbdev ephyr src linux fake
 
 relink:
-	@for i in $(SERVER_SUBDIRS) ; do $(MAKE) -C $$i relink ; done
+	$(AM_V_at)for i in $(SERVER_SUBDIRS) ; do $(MAKE) -C $$i relink ; done
diff --git xorg-server-1.7.99.902/hw/kdrive/ephyr/Makefile.am xorg-server-1.7.99.902/hw/kdrive/ephyr/Makefile.am
index bd74791..d1dd630 100644
--- xorg-server-1.7.99.902/hw/kdrive/ephyr/Makefile.am
+++ xorg-server-1.7.99.902/hw/kdrive/ephyr/Makefile.am
@@ -81,7 +81,7 @@ Xephyr_DEPENDENCIES =	\
 Xephyr_LDFLAGS = $(LD_EXPORT_SYMBOLS_FLAG)
 
 relink:
-	rm -f $(bin_PROGRAMS) && $(MAKE) $(bin_PROGRAMS)
+	$(AM_V_at)rm -f $(bin_PROGRAMS) && $(MAKE) $(bin_PROGRAMS)
 
 MAN_SRCS = Xephyr.man.pre
 
diff --git xorg-server-1.7.99.902/hw/kdrive/fake/Makefile.am xorg-server-1.7.99.902/hw/kdrive/fake/Makefile.am
index 6a5cedf..6d3ed05 100644
--- xorg-server-1.7.99.902/hw/kdrive/fake/Makefile.am
+++ xorg-server-1.7.99.902/hw/kdrive/fake/Makefile.am
@@ -27,4 +27,4 @@ Xfake_DEPENDENCIES =	\
 	@KDRIVE_LOCAL_LIBS@
 
 relink:
-	rm -f $(bin_PROGRAMS) && $(MAKE) $(bin_PROGRAMS)
+	$(AM_V_at)rm -f $(bin_PROGRAMS) && $(MAKE) $(bin_PROGRAMS)
diff --git xorg-server-1.7.99.902/hw/kdrive/fbdev/Makefile.am xorg-server-1.7.99.902/hw/kdrive/fbdev/Makefile.am
index a1664ec..ec9df95 100644
--- xorg-server-1.7.99.902/hw/kdrive/fbdev/Makefile.am
+++ xorg-server-1.7.99.902/hw/kdrive/fbdev/Makefile.am
@@ -25,5 +25,5 @@ Xfbdev_DEPENDENCIES =	\
 Xfbdev_LDFLAGS = $(LD_EXPORT_SYMBOLS_FLAG)
 
 relink:
-	rm -f $(bin_PROGRAMS) && $(MAKE) $(bin_PROGRAMS)
+	$(AM_V_at)rm -f $(bin_PROGRAMS) && $(MAKE) $(bin_PROGRAMS)
 endif
diff --git xorg-server-1.7.99.902/hw/vfb/Makefile.am xorg-server-1.7.99.902/hw/vfb/Makefile.am
index c6601c2..a183824 100644
--- xorg-server-1.7.99.902/hw/vfb/Makefile.am
+++ xorg-server-1.7.99.902/hw/vfb/Makefile.am
@@ -49,4 +49,4 @@ SUFFIXES += .$(APP_MAN_SUFFIX) .man
 EXTRA_DIST = Xvfb.man.pre
 
 relink:
-	rm -f Xvfb$(EXEEXT) && $(MAKE) Xvfb$(EXEEXT)
+	$(AM_V_at)rm -f Xvfb$(EXEEXT) && $(MAKE) Xvfb$(EXEEXT)
diff --git xorg-server-1.7.99.902/hw/xfree86/Makefile.am xorg-server-1.7.99.902/hw/xfree86/Makefile.am
index 914e11f..116113b 100644
--- xorg-server-1.7.99.902/hw/xfree86/Makefile.am
+++ xorg-server-1.7.99.902/hw/xfree86/Makefile.am
@@ -114,7 +114,7 @@ CPP_FILES_FLAGS = \
 	-DMODULEPATH=\"$(DEFAULT_MODULE_PATH)\"
 
 relink:
-	rm -f Xorg && $(MAKE) Xorg
+	$(AM_V_at)rm -f Xorg && $(MAKE) Xorg
 
 xorg.conf.example.pre: xorgconf.cpp
 	cp $(srcdir)/xorgconf.cpp $@
diff --git xorg-server-1.7.99.902/hw/xnest/Makefile.am xorg-server-1.7.99.902/hw/xnest/Makefile.am
index 3e97490..877ab8b 100644
--- xorg-server-1.7.99.902/hw/xnest/Makefile.am
+++ xorg-server-1.7.99.902/hw/xnest/Makefile.am
@@ -89,4 +89,4 @@ SUFFIXES += .$(APP_MAN_SUFFIX) .man
 	$(LN_S) $< $@
 
 relink:
-	rm -f Xnest$(EXEEXT) && $(MAKE) Xnest$(EXEEXT)
+	$(AM_V_at)rm -f Xnest$(EXEEXT) && $(MAKE) Xnest$(EXEEXT)
diff --git xorg-server-1.7.99.902/hw/xwin/Makefile.am xorg-server-1.7.99.902/hw/xwin/Makefile.am
index 8bf6546..41a4515 100644
--- xorg-server-1.7.99.902/hw/xwin/Makefile.am
+++ xorg-server-1.7.99.902/hw/xwin/Makefile.am
@@ -222,4 +222,4 @@ EXTRA_DIST = \
 	xlaunch/window/wizard.h
 
 relink:
-	rm -f XWin$(EXEEXT) && $(MAKE) XWin$(EXEEXT)
+	$(AM_V_at)rm -f XWin$(EXEEXT) && $(MAKE) XWin$(EXEEXT)
-- 
1.7.0.2

